resources/sdk/purecloudjava/templates/ApiClient.mustache

```diff
@@ -672,14 +771,21 @@ public class ApiClient implements AutoCloseable {
             Retry retry = new Retry(retryConfiguration);
             do {
                 connectorResponse = connector.invoke(connectorRequest);
+                // copy unmodifiable maps to make modifiable
+                Map<String,String> requestHeaderCopy = new HashMap<>(request.getHeaderParams());
+                Map<String,String> responseHeaderCopy = new HashMap<>(connectorResponse.getHeaders());
+                logger.debug(request.getMethod(), request.getPath(), (Object)request.getBody(), connectorResponse.getStatusCode(), requestHeaderCopy);
```

Comment:
The only request header I'm seeing is the "Authorization: [REDACTED]" header. There should be a few more headers in the request. Check that your source for the headers is correct

Reply:
Also, if possible try to log the full URL of the path. e.g. https://api.mypurecloud.com/api/v2/users/me instead of /api/v2/users/me

Reply:
Also, I tried making a post request and the request body wasn't logged despite logRequestBody being set to true. I used a debugger and requestBodyToString returned null because neither of the conditions checked in it was met.

Reply:
The request was made using:
```
CreateUser createUser = new CreateUser();
createUser.setName("name");
usersApi.postUsers(createUser);
```

Reply:
Also, I tried calling `notificationsApi.deleteNotificationsChannelSubscriptions("testchannelid");` and it logged the URL as `/api/v2/notifications/channels/{channelId}/subscriptions` instead of `/api/v2/notifications/channels/testchannelid/subscriptions`

Reply:
Made some changes to the sources for the log params, let me know if any of these problems persist.  I'll do my own validation as well.

Reply:
Still not seeing the request body logged, working on that now.